stop-mycroft.sh

```diff
@@ -87,6 +89,13 @@ case ${OPT} in
     end-process skills
     end-process audio
     end-process speech
+
+    # determine platform type
+    mycroft_platform=$( jq -r ".enclosure.platform" < /etc/mycroft/mycroft.conf )
```

Comment:
This errors on a desktop because the file doesn't exist. Can we extract this into a `get_platform` function that checks if the file exists?